Implement user email change functionality (#2290)
DMOJ/online-judge: templates/registration/email_change_notify_email.txt

{{ user.get_username() }},

> {% trans %}You have requested to change your email address to {{ new_email }} for your user account at {{ site_name }}.{% endtrans %}

Reviewer: ```suggestion
{% trans %}Someone (hopefully you!) has requested to change the email address associated with your user account at {{ site_name }} to {{ new_email }}. {% endtrans %}
```
